src/periphery/forge/index.ts

```diff
@@ -7,6 +8,10 @@ export async function deployContract({
   from,
   cwd,
 }: { contractName: string; rpc: string; from: Address; cwd: string }): Promise<Address> {
-  const result = await $`forge create --rpc-url ${rpc} --from ${from} ${contractName} --unlocked --json`.cwd(cwd).json()
+  const result = await $`forge create  --broadcast --rpc-url ${rpc} --from ${from} ${contractName} --unlocked --json`
+    .cwd(cwd)
+    .json()
+  core.info(`Deployed spell ${contractName} to address ${result.deployedTo}`)
+  core.debug(`Deploy result: ${result}`)
```

Comment:
Are u sure that `result` doesn't leak admin RPC for example? I assume that the output is public for public repos so I would remove it.

Reply:
I haven't checked that specifically, but also remember that the warning message that made me realize of the newly added `--broadcast` flag weren't present in the json output
Also given foundry is likely to push breaking changes like the one motivating this PR, even if secrets aren't being leaked now, they could be leaked with future foundry updates.

Good catch, thanks! implemented it in e75276a